index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - Subrat Dash!</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">    
+</head>
+<body>
+    <header>
+        <h2>Welcome</h2>
+        <i class="fa fa-bars fa-2x"></i>
+    </header>
+    <main>
+        <h1>Hey there. I'm <br/>Subrat <br/><span class="profession">I'm a software<br/>developer</span></h1>

```

Comment:
- [ ] according to the requirements the section with the content "I'm a software developer" should be in a `p` tag. please kindly make this small change.
something like this would be fantastic 
```suggestion
        <h1>Hey there. I'm <br/>Subrat <br/><p class="profession">I'm a software<br/>developer</p></h1>

```

Reply:
changed